.github/workflows/pr.yml

```diff
@@ -15,7 +14,15 @@ jobs:
         with:
           access_token: ${{ github.token }}
 
+      - name: Check is merge PR
+        id: check-is-merge
+        run: |
+          if [[ """${{ github.event.pull_request.body }}""" =~ \|?[[:space:]]*Category\?[[:space:]]*\|[[:space:]]*ME[[:space:]]+ ]]; then
```

Comment:
I have no idea what this is doing 🤣

Reply:
That's a bash regex equivalent of the regex here: https://github.com/PrestaShop/prestonbot/blob/master/src/AppBundle/PullRequests/BodyParser.php#L12 with "Category" et "ME" as %s arguments.

Reply:
I like your `with "Category" et "ME"` xD My brain is melting thanks to you xD